File: VehicleTransport/主版本/3.开发源代码/DB_VehicleTransportManage/Model/m_Plan_Load.cs
/**  版本信息模板在安装目录下，可自行修改。
* m_Plan_Load.cs
*
* 功 能： N/A
* 类 名： m_Plan_Load
*
* Ver    变更日期             负责人  变更内容
* ───────────────────────────────────
* V0.01  2014-08-11 11:08:35   N/A    初版
*
* Copyright (c) 2012 Maticsoft Corporation. All rights reserved.
*┌──────────────────────────────────┐
*│　此技术信息为本公司机密信息，未经本公司书面同意禁止向第三方披露．　│
*│　版权所有：动软卓越（北京）科技有限公司　　　　　　　　　　　　　　│
*└──────────────────────────────────┘
*/
using System;
namespace DB_VehicleTransportManage.Model
{
	/// <summary>
	/// m_Plan_Load:实体类(属性说明自动提取数据库字段的描述信息)
	/// </summary>
	[Serializable]
	public partial class m_Plan_Load
	{
		public m_Plan_Load()
		{}
		#region Model
		private int _id;
		private int? _planid;
		private int? _vehicleid;
		private int? _materietypeid;
		private decimal? _n_count;
		private string _vc_memo;
		private int? _i_flag=0;
		/// <summary>
		/// 
		/// </summary>
		public int ID
		{
			set{ _id=value;}
			get{return _id;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int? PlanID
		{
			set{ _planid=value;}
			get{return _planid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int? VehicleID
		{
			set{ _vehicleid=value;}
			get{return _vehicleid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int? MaterieTypeID
		{
			set{ _materietypeid=value;}
			get{return _materietypeid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public decimal? n_Count
		{
			set{ _n_count=value;}
			get{return _n_count;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string vc_Memo
		{
			set{ _vc_memo=value;}
			get{return _vc_memo;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int? i_Flag
		{
			set{ _i_flag=value;}
			get{return _i_flag;}
		}
		#endregion Model

	}
}

File: VehicleTransport/主版本/3.开发源代码/客户端/手机客户端/Last of KJ915_bestway/src/com/bestway/kj915/domain/req/load/m_Plan_Load.java
package com.bestway.kj915.domain.req.load;

public class m_Plan_Load {
	public int ID;//¸ø0¾ÍÐÐ
	public int PlanID=0;

	public int VehicleID;
	public int MaterieTypeID;

	public int n_Count;

	public String vc_Memo;

	public int i_Flag;
	
	

	public m_Plan_Load() {
		super();
	}



	public m_Plan_Load(int iD, int planID, int vehicleID, int materieTypeID,
			int n_Count, String vc_Memo, int i_Flag) {
		super();
		ID = iD;
		PlanID = planID;
		VehicleID = vehicleID;
		MaterieTypeID = materieTypeID;
		this.n_Count = n_Count;
		this.vc_Memo = vc_Memo;
		this.i_Flag = i_Flag;
	}
	
	
}
